book.rb

```diff
@@ -0,0 +1,14 @@
+class Book
+  attr_accessor :title, :author, :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(rental)
+    @rentals << rental
+    rental.book = self
```

Comment:
- [ ] nice work adding the rental method. you know it includes the person, the book and rental date. please note that the book itself should be sent as a parameter to create the new rental achieving the 'has-many' association.` you should have something like

```ruby
def add_rental(person, date)
    Rental.new(date, self, person)
  end
```